### DISCUSSION TITLE
Reverse proxying with nginx and "Authorization header doesn't match"

### DISCUSSION THREAD

**Discussion by ccantill** [structurizr/lite#129] [Q&A]
I'm trying to put Structurizr lite (deployed using docker) behind a reverse proxy using nginx. I'm mapping `https://acme.com/structurizr/` to `http://localhost:8080/` (replaced actual hostname with acme.com). I can access `https://acme.com/structurizr/workspace/diagrams` which presents the UI just fine, but shows an error "Authorization header doesn't match", coming from a XHR call to `https://acme.com/structurizr/api/workspace/1?_=1713440682875` 
I've set `structurizr.url=https://acme.com/structurizr/` in `/usr/local/structurizr/structurizr.properties`

**Comment by simonbrowndotje** [comment]
Lite is only designed to work as the root application I'm afraid (i.e. `https://acme.com` rather than `https://acme.com/structurizr`).
